### PR TITLE
Allowed DoctrineMigrationsBundle to work with Symfony apps using DBAL only

### PR DESCRIPTION
**Pull request by guilhermeblanco**
I also fixed a bug of accepting optional `--em` value (it should be required).

### CODE REVIEW

**Comment by mikeSimonson** [comment]
Thanks @guilhermeblanco 
Can you add tests ?

**Comment by guilhermeblanco** [comment]
@mikeSimonson this bundle doesn't even have infrastructure for tests yet. That would be a task for another PR, not this one. =)

**Comment by mikeSimonson** [comment]
@guilhermeblanco What do you mean the em should be required. I think it's only required for the Diff command.

**Comment by guilhermeblanco** [comment]
@mikeSimonson sorry, I probably had to explain better.
Before it was allowed to run this:

```bash
php app/console doctrine:migrations:diff --em
```

Without requiring a value for `em`.

**Comment by guilhermeblanco** [comment]
@ryancastle I already merged a fix for that from @xabbuh

File: Command/Helper/DoctrineCommandHelper.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ * Provides some helper and convenience methods to configure doctrine commands in the context of bundles
+ * and multiple connections/entity managers.
+ *
+ * @author Guilherme Blanco <guilhermeblanco@hotmail.com>
+ */
+abstract class DoctrineCommandHelper extends BaseDoctrineCommandHelper
+{
+    public static function setApplicationHelper(Application $application, InputInterface $input)
```
Comment (stof) [inline Command/Helper/DoctrineCommandHelper.php]: the naming is weird. It is not setting an application helper

**Pull request merged**